src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -1364,6 +1364,9 @@ public static SqlConnectionStringBuilder CreateConnectionStringBuilder(Connectio
                     case "HGS":
                         connectionBuilder.AttestationProtocol = SqlConnectionAttestationProtocol.HGS;
                         break;
+                    case "None":

```

Comment:
What about using [Enum.TryParse](https://learn.microsoft.com/en-us/dotnet/api/system.enum.tryparse?view=net-7.0) instead?

Reply:
I'll get it done in another PR (wip)